packages/alfa-dom/src/h.ts

```diff
@@ -37,9 +38,10 @@ export function h<N extends string = string>(
   attributes?: Array<Attribute> | Record<string, string | boolean>,
   children?: Array<Node | string>,
   style?: Array<Declaration> | Record<string, string>,
-  box?: Rectangle
+  box?: Rectangle,
+  device?: Device
```

Comment:
**Suggestion:** This (and the same in other functions and in JSX) can default to `Device.standard()`, that way we don't need to provide a device in tests that also use `Device.standard()` by default.